src/ApiManagement/ApiManagement/Commands/NewAzureApiManagementCustomHostnameConfiguration.cs

```diff
@@ -56,6 +56,13 @@ public class NewAzureApiManagementCustomHostnameConfiguration : AzureRMCmdlet
             HelpMessage = "KeyVaultId to the secret storing the Custom SSL Certificate.")]
         public string KeyVaultId { get; set; }
 
+        [Parameter(
+            ValueFromPipelineByPropertyName = false,
+            ParameterSetName = SslCertificateFromKeyVault,
+            Mandatory = false,
+            HelpMessage = "User Assigned Managed Identity ClientId credentials to authenticate to KeyVault to fetch Custom SSL Certificate.")]
```

Comment:
```suggestion
            HelpMessage = "User-assigned managed identity client ID to authenticate to KeyVault to fetch Custom SSL Certificate.")]
```
Client ID is not credential so I suggest remove that word

Reply:
#resolve